src/Database/LSMTree/Internal/Run/BloomFilter.hs

```diff
@@ -39,6 +39,7 @@ module Database.LSMTree.Internal.Run.BloomFilter (
     -- * Bloom
   , Bloom
   , Bloom.elem
+  , Bloom.elemMany
```

Comment:
`elem` many should be rather defined in this (an lsm-tree specific module).

Reply:
As noted above, the fast versions need access to the internal representation.

Reply:
.. which is exported. E.g. the serialization uses it (and doesn't live in `bloomfilter`).